CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## v0.7.1 (2023-03-14)
+
+### New features and changes
```

Comment:
As this is only a bug fix release I would expect something like `fixes` here but this heading aligns with the old one and I am not familiar with recommended headings here, so we can leave it as is.